src/main/java/jenkins/plugins/logstash/LogstashWriter.java

```diff
@@ -82,7 +93,7 @@ public LogstashWriter(Run<?, ?> run, OutputStream error, TaskListener listener,
    *
    * @return the charset
    */
-  public Charset getCharset()
+  public String getCharset()
```

Comment:
is this method even used? maybe remove it

Reply:
yes it is used in LogstashOutputStream

Reply:
it's weird that you changed the type but didn't had to update the LogstashOutputStream
I guess it's passing the value somewhere else, that's why

Reply:
the LogstashOutputStream calls the method and directly passes it to Strings getBytes. This has 2 signatures one with string and one with charset that's why there is no change there.